src/canvas/static_canvas.class.ts

```diff
@@ -1500,10 +1493,9 @@ export class StaticCanvas<
    * @param {string[]} [properties] Array of properties to include in the cloned canvas and children
    * @returns {Promise<Canvas | StaticCanvas>}
    */
-  clone(properties: string[]): Promise<this> {
+  clone(properties: string[]): Promise<StaticCanvas> {
```

Comment:
Should i do a declare with this one on the EventedCanvas? Promise<this> doesn't work.

Reply:
can you explain why it isn't working with this?

Reply:
There is a typescript error that complains that this isn't what he wants, i don't have it, but i can retrigger it and cpy paste it next time i work on this pr